gemrb/core/GameScript/Actions.cpp

```diff
@@ -2625,7 +2633,7 @@ void GameScript::ToggleDoor(Scriptable* Sender, Action* parameters)
 			door->AddTrigger(TriggerEntry(trigger_failedtoopen, actor->GetGlobalID()));
 
 			//playsound unsuccessful opening of door
-			core->PlaySound(door->IsOpen() ? DS_CLOSE_FAIL : DS_OPEN_FAIL, SFXChannel::Actions, *p, GEM_SND_SPATIAL);
+			core->GetAudioPlayback().PlayDefaultSound(door->IsOpen() ? DS_CLOSE_FAIL : DS_OPEN_FAIL, SFXChannel::Actions);
```

Comment:
This should remain spatial, unfortunately not all default sounds are the same. For example the next one about xp shouldn't be.

Reply:
I will think of something.

Reply:
For some context, these are sounds that were hardcoded in the originals. Some are GUI related like the XP etc., but not all. The GUI ones should be global, things like opening and closing sounds spatial.